src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -464,9 +434,6 @@ DictionaryModal.propTypes = {
   modalhide: PropTypes.func.isRequired,
   defaultLocaleOption: PropTypes.object,
   isEditingDictionary: PropTypes.bool,
-  searchDictionaries: PropTypes.func.isRequired,
```

Comment:
Was this also removed from the `mapStateToProps`?

Reply:
Done,,,in mapDispatchToProps